front_end/html/edit_problem.html

```diff
@@ -65,6 +65,9 @@ <h2>Create Problem</h2>
 
     <p><strong>Answer code*: </strong><br />
     <textarea class="textarea is-info is-normal is-medium is-fullwidth" placeholder="Please provide answer code." rows="20" name="answer_code" style="font-family:monospace">{{ problem_details["answer_code"] }}</textarea></p>
+    
+    <p><strong>Answer Description:</strong></string></p>  
```

Comment:
This is nitpicky, but please use sentence casing to be consistent. In this case, that means to use a lower-case "d" on description.